lib/hooks/use-dispatch-event.js

```diff
@@ -1,4 +1,6 @@
-import { hook, Hook } from 'haunted';
+import {
+	hook, Hook
```

Comment:
Hmm this does not look good to me. I think the curly-newline rule should be adjusted for imports. Maybe we need to enable `minProperties` and `consistent` for it? Also it looks a bit weird when there are multiple props per line ?

Reply:
Listing items like this is how it has been done in frontend too. Each item could be on a new line, but that would then use more lines which is also an eslint constraint - to keep the line quantity as low as possible. It feels usable to me as long as the items are alphabetically sorted and the line does not exceed 80 or 160 characters .